src/actor/mod.rs

```diff
@@ -344,14 +367,24 @@ mod tests {
         let tx = spawn_actor();
 
         // create an account
-        let account = AccountToCreate::default();
+        let account = AccountToCreate {
+            client_options: ClientOptionsBuilder::node("http://node.iota").unwrap().build(),
+            mnemonic: None,
+            alias: None,
+            created_at: None,
+        };
         send_message(&tx, MessageType::SetStrongholdPassword("password".to_string())).await;
         let response = send_message(&tx, MessageType::CreateAccount(account)).await;
         match response.response() {
             ResponseType::CreatedAccount(created_account) => {
-                // remove the created account
-                let response = send_message(&tx, MessageType::RemoveAccount(created_account.id().clone())).await;
-                assert!(matches!(response.response(), ResponseType::RemovedAccount(_)));
+                let id = created_account.id().clone();
+                std::thread::spawn(move || {
```

Comment:
Why spawn a thread here? If there's an async executor running it would be better to just create an async task

Reply:
this is just a test case that will be changed soon